java/README.md

```diff
@@ -130,6 +130,22 @@ Maven:
 </dependency>
 ```
 
+SBT:
+- **IMPORTANT** must include a `classifier`. Please use this dependency block and add it to the build.sbt file.
+```scala
+// osx-aarch_64
+libraryDependencies += "io.valkey" % "valkey-glide" % "1.0.1" classifier "osx-aarch_64"
+
+// osx-x86_64
+libraryDependencies += "io.valkey" % "valkey-glide" % "1.0.0" classifier "osx-x86_64"
+
+// linux-aarch_64
+libraryDependencies += "io.valkey" % "valkey-glide" % "1.0.0" classifier "linux-aarch_64"
```

Comment:
it's already an issue - above you wrote 1.0.1 and here 1.0.0. change it so we wont have hardcoded versions. and it's a must, use a variable once and set it only in one place.

Reply:
For the documentation it does not make sense to use a variable.  Every user will copy a different line depending on their platform and only one line.  I will replace these all to use a wildcard.